app/models.py

```diff
@@ -480,6 +480,15 @@ class ReplyToInbox(db.Model):
     updated_at = db.Column(db.DateTime, nullable=True, onupdate=datetime.datetime.utcnow)
 
 
+class Session(db.Model):
+    __tablename__ = "sessions"
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
```

Comment:
I'm not going to request changes, but `nullable=False` is redundant for a primary key.